tests/zfs-tests/tests/functional/resilver/resilver_restart_001.ksh

```diff
@@ -47,12 +47,13 @@ function cleanup
 {
 	echo $ORIG_RESILVER_MIN_TIME > $ZFS_PARAMS/zfs_resilver_min_time_ms
 	echo $ORIG_SCAN_SUSPEND_PROGRESS > $ZFS_PARAMS/zfs_scan_suspend_progress
+	echo $ORIG_ZFS_ZEVENT_LEN_MAX > $ZFS_PARAMS/zfs_zevent_len_max
```

Comment:
It looks like this was overlooked in the initial review, but there are wrapper functions to set these tuning portably.  Can you update this PR to use `set_tunable64` and `get_tunable` for all of the modified tunings.  See `zpool_wait/zpool_wait_multiple.ksh` for an example.  But in short:

```sh
ORIG_ZFS_ZEVENT_LEN_MAX=$(get_tunable zfs_zevent_len_max)
log_must set_tunable64 zfs_zevent_len_max 512

...

log_must set_tunable64 zfs_zevent_len_max $ORIG_ZFS_ZEVENT_LEN_MAX
```

Reply:
Done